filament/include/filament/IndirectLight.h

```diff
@@ -281,6 +281,21 @@ class UTILS_PUBLIC IndirectLight : public FilamentAPI {
      */
     const math::mat3f& getRotation() const noexcept;
 
+    /**
+     * Specifies the rotation around the Z axis to apply to the IBL. Performs rotation in shaders,
+     * without applying a root transformation to the scene.
+     *
+     * @param rotation Angle value in radians.
+     *
+     * @return This Builder, for chaining calls.
+     */
+    void setRotationZ(float rotation) noexcept;
```

Comment:
Any reason why a new interface was introduced, quite confusingly living alongside a `get/setRotation(mat3f)` interface? I mean the implementation behind the existing interface could be turned into a uniform + shader operation, replacing the root transform solution.

Reply:
I wanted to add to the interface instead of modifying the implementation, to keep our changes separate from upstream. I don't have a strong opinion about this, so I could change the implementation too.

Reply:
After our discussion, I made the modifications in 305b102